src/vm/vm_memory/memory.rs

```diff
@@ -84,6 +86,17 @@ impl Memory {
         }
     }
 
+    //Gets the value from memory address.
+    //If the value is an MaybeRelocatable::Int(Bigint) return &Bigint
+    //else raises Err
+    pub fn get_integer(&self, key: &MaybeRelocatable) -> Result<&BigInt, VirtualMachineError> {
```

Comment:
I wasn't sure whether to add this function, to the Memory module or to HintsUtils one.
I leave it here because I think we can use it in the future for other things apart from hints.

Reply:
Yes, I think it's more appropriate here. The sooner we can specialize types without going through hoops, the better.

Reply:
Shouldn't the argument be a `Relocatable`?